debian/changelog

```diff
@@ -1,5 +1,8 @@
 python-oq-risklib (0.6.0-1) precise; urgency=low
 
+    [Michele Simionato]
```

Comment:
You are right, the strange indentation came from the resolution of a merge conflict